easybuild/easyconfigs/l/libyaml/libyaml-0.2.2-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,32 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Nils Christian <nils.christian@uni.lu>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'libyaml'
```

Comment:
@Flamefire it looks like `LibYAML` is the official software name (with capitalization). please change accordingly?

Reply:
Note that the old packaging we have is called libyaml

Reply:
I left it lowercase for consistency. Do we have to rename the folder and filename too? Is it worth it?

Reply:
@akesandgren ah, I missed that. better to keep the old name then.